api/envoy/extensions/cache/simple_http_cache/v3/config.proto

```diff
@@ -1,10 +1,10 @@
 syntax = "proto3";
 
-package envoy.extensions.cache.simple_http_cache.v3alpha;
+package envoy.extensions.cache.simple_http_cache.v3;
```

Comment:
I think it's totally valid to break stuff since these are alpha APIs. One alternative we have though is to clone v3alpha to v3 and just not support v3alpha in Envoy. I'm not sure how useful that would be, but if other API clients depend on alpha protos, that could be less bad. @markdroth any opinions here?

Reply:
I don't have any strong opinion on this.  gRPC doesn't use any alpha APIs, and I don't think that GCP depends on any of these APIs either.

In principle, I guess I agree that not forcibly breaking anyone who is depending on the alpha protos might be a bit more user-friendly, but at the same time, the whole point of alpha is to be able to make breaking changes, so... I can see both sides of this.

Reply:
Unless there are strong objections I'm just going to suggest we pull the bandaid here and get it done. If we receive extreme complaining we could potentially add the files back but let's see what happens?

Reply:
That sounds fine to me.